CMakeLists.txt

```diff
@@ -6,8 +6,16 @@ include(GNUInstallDirs)
 include(Utils)
 include(third_party/get_third_party)
 
+# Should update cmake to a more recent version which supports FindPython3.

```

Comment:
The CMake build system is optional.  Should way say that it's required when using TensorFlow?

Reply:
No, TensorFlow only officially supports Bazel. We are free to use whatever we want.

So far I choose CMake because it is easier on my macbook - where many build tools or libraries on Linux are absent here.

Reply:
I just meant when using TensorFlow with Kaldi.  It's OK, not everyone will use this, so being limited to the CMake build setup is OK.